### PR TITLE
[Wallet] Visual refinement 2

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

1 set an explicit weight of normal on our regular font as without one in places where a third part component set a default weight it would not be overridden. example the drawer item component. 

2. use small not regular for the `PhoneNumbeWithEmoji` (Cory confirmed this was desired in both places it is used)

3. fix extra space at top of screens caused by safe area view. also

4. place exchange confirmation view's scroll view instead of the safe area view instead of other way to match how we have other screens set. 

5. hamburger in top bar was not centered vertically and was too far to the right. 

### Tested

Screen Shot 2020-07-16 at 1 41 45 PM
Screen Shot 2020-07-16 at 3 02 25 PM
Screen Shot 2020-07-16 at 3 02 33 PM
Screen Shot 2020-07-16 at 3 02 38 PM
Screen Shot 2020-07-16 at 3 02 46 PM
Screen Shot 2020-07-16 at 3 02 52 PM
Screen Shot 2020-07-16 at 3 51 44 PM

### Related issues

- Fixes a few bugs on the wallet design bug list

### CODE REVIEW

File: packages/mobile/src/navigator/DrawerNavigator.tsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ export default function DrawerNavigator() {
       drawerContent={drawerContent}
       backBehavior={'initialRoute'}
       drawerContentOptions={{
-        labelStyle: [fontStyles.regular, { marginLeft: -20 }],
```
Comment (aaronmgdr) [inline packages/mobile/src/navigator/DrawerNavigator.tsx:188]: without setting weight to normal we get 500 set by react-navigation.

note the margin -20 was here before

Comment (aaronmgdr) [reply packages/mobile/src/navigator/DrawerNavigator.tsx:188]: i also considered setting theis globaly, but the snapshot diff was crazy big so i figured lets be conservative

File: packages/mobile/src/exchange/ExchangeConfirmationCard.tsx [hunk 2]
```diff
@@ -101,8 +101,8 @@ export default function ExchangeConfirmationCard(props: Props) {
             <TotalLineItem amount={totalAmount} />
           </View>
         </View>
-      </SafeAreaView>
-    </ScrollView>
+      </ScrollView>
+    </SafeAreaView>
```
Comment (jeanregisser) [inline packages/mobile/src/exchange/ExchangeConfirmationCard.tsx]: `SafeAreaView` should generally be inside `ScrollView` so it displays nicely on devices with a notch.
Any reason not to do that here?

File: packages/react-components/styles/fonts.v2.tsx [hunk 3]
```diff
@@ -79,7 +79,7 @@ const fontStyles = StyleSheet.create({
     color: colors.dark,
   },
   large: standards.large,
-  regular: standards.regular,
+  regular: { ...standards.regular },
```
Comment (jeanregisser) [inline packages/react-components/styles/fonts.v2.tsx]: Why is this needed?

**Review by jeanregisser — COMMENTED** [review]
Thanks for these!! 👍 🎉 
Left some very small questions.

File: packages/react-components/styles/fonts.v2.tsx (hunk 3 above)
Comment (aaronmgdr) [reply packages/react-components/styles/fonts.v2.tsx]: ah leftover from when i tried adding font weight normal here.

File: packages/mobile/src/exchange/ExchangeConfirmationCard.tsx (hunk 2 above)
Comment (aaronmgdr) [reply packages/mobile/src/exchange/ExchangeConfirmationCard.tsx]: ah ha I got confused and thought it was the other way

Comment (aaronmgdr) [reply packages/mobile/src/exchange/ExchangeConfirmationCard.tsx]: Looking at the code I see 2 places where SafeAreaView is inside the scroll view and  about 10+ where it is outside.

Comment (jeanregisser) [reply packages/mobile/src/exchange/ExchangeConfirmationCard.tsx]: Yes when I initially added support for iOS, I put all of them outside as a shortcut.
In the long term most of them are meant to be inside, except when there's a fixed button area at the bottom of the screen.

**Review by jeanregisser — APPROVED** [review]
Great 👍
